lab5.py

```diff
@@ -0,0 +1,156 @@
+"""
+lab5.py_corrected.py
+"""
+
+
+class RaceHorse:
+    """
+    class
+    """
+
+    def __init__(self, speed, age, name, place_in_race=None):
+        """
+        init
+        :param speed:
+        :param age:
+        :param name:
+        :param place_in_race:
+        """
+        self.__speed = speed
+        self.__age = age
+        self.__name = name
+        self.__place_in_race = place_in_race
+
+    def get_speed(self):
+        """
+        get_speed
+        :return:

```

Comment:
:return: The speed of the horse
```suggestion
       :return: the speed of the horse
```